test/SIL.XForge.Tests/Realtime/ConnectionTests.cs

```diff
@@ -408,7 +408,7 @@ public async Task SubmitOpAsync_UsesUnderlyingRealtimeServerOutsideOfATransactio
             List<Json0Op> op = builder.Op;
 
             // SUT
-            var result = await env.Service.SubmitOpAsync(collection, id, op, snapshot.Data, snapshot.Version);
+            _ = await env.Service.SubmitOpAsync(collection, id, op, snapshot.Data, snapshot.Version);
```

Comment:
Is it required to assign the value when awaiting? I think in TS we frequently just don't assign the returned value.

Reply:
You are right, it is not necessary. Sometimes I will use this pattern to make sure it is clear in the test that the return value is ignored, but it is not done elsewhere. Removed.